### PR TITLE
Upgrade to macos-12 in native builds CI

### PR DESCRIPTION
**Pull request by thomas-bc**
Fix https://github.com/nasa/fpp/issues/456

Hopefully this should be all that is needed, but I am not sure of if there could be side effects or incompatibilities.
We may want to wait for @LeStarch to confirm as he is much more familiar with the native build process than me.

### CODE REVIEW

**Comment by bocchino** [comment]
Thanks, Thomas! This change looks good to me.

File: .github/workflows/native-build.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ jobs:
           matrix = {
             "run": [
               {
-                "runner": "macos-11",
+                "runner": "macos-12",
                 "tag": "macosx_10_9_universal2"
```
Comment (LeStarch) [inline .github/workflows/native-build.yml:105]: Doesn't this tag need to change too?  New image means new build means new osx minimum version....

**Review by LeStarch — COMMENTED** [review]
One question.

**Review by LeStarch — APPROVED** [review]
Fixed the tag for you.  Technically we could automate this step....but not sure it is worth the finesse over a few version points.

**Comment by bocchino** [comment]
Thanks, Michael!

**Pull request merged**